Dockerfile

```diff
@@ -259,6 +259,9 @@ RUN pip3 install pango_aliasor==0.3.0
 RUN pip3 install pathogen-embed==3.0.0
 RUN pip3 install xlrd==2.0.1
 
+# Install openpyxl for pandas in GenoFLU
+RUN pip3 install openpyxl==3.1.0
```

Comment:
Huh - what about GenoFLU makes this needed? (I'm sure it is needed, I'm just curious). I see it as a dependency in the GenoFLU conda env but don't see it imported by any of the (n=2) python scripts.

Reply:
Not used directly in GenoFLU, but openpyxl needed for `pandas.read_excel`. 

I can also do `pip3 install pandas[excel]` so this is clearer.

Reply:
Gotcha. We may end up using openpyxl somewhere else so probably clearer to leave as-is?